monkey/common/common_consts/telem_categories.py

```diff
@@ -7,6 +7,5 @@ class TelemCategoryEnum:
     POST_BREACH = "post_breach"
     SCAN = "scan"
     STATE = "state"
-    SYSTEM_INFO = "system_info"
```

Comment:
Looks like I also left some references in `example.conf`, I wonder if we even need that file